src/sentry/runner/commands/devserver.py

```diff
@@ -35,6 +36,9 @@ def _get_daemon(name):
 @click.option(
     "--prefix/--no-prefix", default=True, help="Show the service name prefix and timestamp"
 )
+@click.option(
+    "--pretty/--no-pretty", default=True, help="Styleize various outputs from the devserver"
```

Comment:
```suggestion
    "--pretty/--no-pretty", default=True, help="Stylize various outputs from the devserver"
```

Reply:
Also, might be worth detecting TTY for the default value.

Reply:
Oop, this should probably not default to true. 🤔

Reply:
I'd say it should default to `true` for interactive terminals that support color. We don't have to detect this with 100% accuracy, just a cursory sniff should be enough to cover most of the cases.